package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "bnc-onboard",
-  "version": "1.38.4",
+  "version": "1.38.4-0.0.1",
```

```suggestion
  "version": "1.39.0",
```
We can push a minor version bump for this change, no need to add the staging version at the end

---

Changes implemented.